WcaOnRails/db/migrate/20200123042042_markfeetasunofficial.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class MarkFeetAsUnofficial < ActiveRecord::Migration[5.2]
+  def change
+    Event.find("333ft").update!(rank: 996)
```

Comment:
`update!` operations are not reversible, therefore you cannot use them in a `change` method of a migration.
Here the appropriate way to do this would have been to do an `up` updating to 996, and a `down` updating to 100.

Reply:
thanks, I will try instead using the up and down methods now

Reply:
Thanks. Given that there are actually quite some other changes needed I've went ahead and submitted #5099.
Also creating the migration is only one step, you also have to run it locally and commit the changes introduced to the database structure (see [here](https://github.com/thewca/worldcubeassociation.org/pull/5099/files#diff-cec4526bf4005aa66536d776767d1d81L1628)).